pkg/util/templates/templates.go

```diff
@@ -162,3 +169,13 @@ func sampleTemplateLabelsWithSev(url, severity string) map[string]string {
 		"RouteMonitorUrl": url,
 	}
 }
+
+func GetClusterID(c client.Client) string {
+	var version configv1.ClusterVersion
+	fmt.Println("Called GetClusterID")
+	err := c.Get(context.TODO(), client.ObjectKey{Name: "version"}, &version)
+	if err != nil {
+		return ""
+	}
```

Comment:
Shouldn't we try getting the ID again with the next reconcile instead of ignoring the error?

Reply:
For the case something fails, it is set to an empty string, and it will try to fetch it again in the next reconcile. However, there is actually not a big chance that this fails as long as API is up and Permissions are set

Reply:
I _guess_ there could be more and uninteresting reasons the k8s api call could fail